src/webpack.common.js

```diff
@@ -24,7 +24,6 @@ module.exports = {
 			{
 				test: /\.js$/,
 				loader: 'babel-loader',
-				exclude: /node_modules/,
```

Comment:
That means it will transpile everything.
It will drastically increase the compile process time :)

Reply:
Should be ok though as the app is not that big

Reply:
Ah, that's why it was so slow all of a sudden. So let me use `babel-loader-exclude-node-modules-except` again.